Src/BasicPostProcess.cpp

```diff
@@ -592,4 +594,4 @@ void BasicPostProcess::SetBloomBlurParameters(bool horizontal, float size, float
     pImpl->bloomBrightness = brightness;
     pImpl->bloomHorizontal = horizontal;
     pImpl->SetDirtyFlag();
-}
```

Comment:
``-Wnewline-eof`` which would also be gripped about with ``-Wc++98-compat-pedantic``